eng/common/scripts/common.ps1

```diff
@@ -63,6 +63,7 @@ $GetEmitterNameFn = "Get-${Language}-EmitterName"
 $GetDirectoriesForGenerationFn = "Get-${Language}-DirectoriesForGeneration"
 $UpdateGeneratedSdksFn = "Update-${Language}-GeneratedSdks"
 $IsApiviewStatusCheckRequiredFn = "Get-${Language}-ApiviewStatusCheckRequirement" 
+$DependentPackagesFromPackageSetFn = "Get-${Language}-DependentPackagesFromPackageSet"
```

Comment:
Whatever term we end up using for the PackageProperties (i.e. Canary or whatever) we should name this function similarly.

Reply:
Updated and aligned.